aws-athena/ctas/proximity-dist_pin_to_airport.sql

```diff
@@ -0,0 +1,81 @@
+-- CTAS to create a table of distance to the airports for each PIN
+
+{{
+  config(
+    materialized='table',
+    partitioned_by=['year'],
+    bucketed_by=['pin10'],
+    bucket_count=1
+  )
+}}
+
+WITH distinct_pins AS (
+    SELECT DISTINCT
+        x_3435,
+        y_3435
+    FROM {{ source('spatial', 'parcel') }}
+),
+
+xy_to_airports_dist AS (
+    SELECT
+        dp.x_3435,
+        dp.y_3435,
+        ST_DISTANCE(
+            ST_POINT(dp.x_3435, dp.y_3435),
+            ST_POINT(1099852, 1935070) --ohare centroid
+        ) AS dist_to_ohare,
+        ST_DISTANCE(
+            ST_POINT(dp.x_3435, dp.y_3435),
+            ST_POINT(1142843, 1864827) --midway centroid
+        ) AS dist_to_midway        
+    FROM distinct_pins AS dp
+),
+
+--COEFFICIENTS FROM OMP-ONLY inverse square noise falloff model for O'Hare,
+-- 2019-sensor data ONLY inverse square noise falloff model for Midway:
+--ohare     0.00582564978995262
+--midway    0.00297149980129393
```

Comment:
**suggestion:** Let's link to the R script on GitHub that generates these coefficients (once #193 is merged).